vocabularies/georesources-report-types.ttl

```diff
@@ -358,6 +358,35 @@ grt:permit-report-surrender a skos:Concept ;
     skos:prefLabel "Coal or Mineral Permit Report - Surrender"@en ;
     skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
 
+grt:petroleum-end-authority a skos:Concept ;
+    skos:altLabel "End of Authority Report (DAA)"@en,
+        "Petroleum Report - End of Authority Report"@en;
+    skos:definition "A report that is required to be lodged at the end of tenure over a Data Acquisition Authority, detailing the work undertaken on the permit throughout the life of tenure."@en  ;
+    skos:inScheme <http://linked.data.gov.au/def/georesource-report> ;
+    skos:notation "PETAUT" ;
+    skos:prefLabel "Petroleum Report - End of Authority (DAA)"@en ;
+    skos:topConceptOf <http://linked.data.gov.au/def/georesource-report> .
+
+grt:petroleum-end-tenure a skos:Concept ;
+    skos:altLabel "End of Tenure Report (ATP)"@en,
+        "End of Tenure Report (PL)"@en,
+        "Petroleum Report - End of Tenure Report"@en,
+        "Petroleum Report - Final Report"@en,
+        "Petroleum Report - End of Tenure"@en,
+        "Petroleum Report - Final"@en;

```

Comment:
Is Final not the preflabel?

Reply:
See suggestion below (at bottom of comments)